server/src/main/java/org/apache/iotdb/db/mpp/plan/planner/LocalExecutionPlanner.java

```diff
@@ -138,10 +138,12 @@ public static LocalExecutionPlanner getInstance() {
   public DataDriver plan(
       PlanNode plan,
       TypeProvider types,
+      Map<String, List<Integer>> deviceToMeasurementIndexesMap,
```

Comment:
You could put this field into DeviceMergeNode.

Reply:
fixed